rmw_dds_common/include/rmw_dds_common/context.hpp

```diff
@@ -27,16 +27,27 @@
 namespace rmw_dds_common
 {
 
+/// Structure of data that a Context will need in any rmw implementation
+/// mapping one Participant to Multiple Nodes.
 struct Context
 {
+  /// Gid of the Participant that the context uses.
   rmw_gid_t gid;
+  /// Publisher used to publish ParticipantEntitiesInfo discovery data.
   rmw_publisher_t * pub;
+  /// Subscriber used to listen ParticipantEntitiesInfo discovery data.
   rmw_subscription_t * sub;
+  /// Cached graph from listened discovery data.
   GraphCache graph_cache;
+  /// Mutex that should be locked when updating graph cache/publishing a message.
```

Comment:
@ahcorde @ivanpauno nit:
```suggestion
  /// Mutex that should be locked when updating the graph cache...
```
I'm unsure as to how to rephrase. Publish what message?

Reply:
rephrased, see https://github.com/ros2/rmw_dds_common/pull/26/commits/462c77547848704b78218103ae068db6a666e728